packages/moonstone/internal/Picker/tests/Picker-specs.js

```diff
@@ -215,6 +215,7 @@ describe('Picker Specs', function () {
 
 		const expected = -1;
 		picker.simulate('keyDown', {keyCode: 37});
+		picker.simulate('mousedown');
```

Comment:
Note for reviewer: The proposed change to `Spottable` will now allow for the `mousedown` event to be forwarded/simulated when specified keys (like in this case, the left arrow key) are pressed. In practice, there is no change in behavior from the previous implementation. Strangely, in these tests it appears we must now explicitly call `picker.simulate('mousedown')` even though I feel this should be included via the `keydown` event in the prior line.

The `keydown` event still sets the internal flag in `Picker` that designates how the value should change based on which direction was pressed, so the test is still valid. That being said, I'm wondering if there is a reason we must explicitly call this line now and if there is a way to avoid this. Thoughts?